### PR TITLE
Add DL features to documents 

### PR DESCRIPTION
**Pull request by parisa-zahedi**

### CODE REVIEW

File: bioacoustics/feature_extraction/README.md [hunk 1]
```diff
@@ -29,8 +29,35 @@ Use shell script `run.sh` to start `main.py` from the command line. The followin
...
 In `./config` the user can specify which features to extract.
 
-## sndfile library
+### sndfile library
 If you get an error saying something about a 'snd_file' dependency on an ubuntu machine, this can be fixed by installing the following C library:
 ```
 sudo apt-get install libsndfile-dev
 ```
+## Feature extraction for Convolutional Neural Network (CNN)
+To extract audio features for CNN classifier, .wav files are converted to Log-mel Spectrograms using [librosa](https://zenodo.org/badge/latestdoi/6309729) library.
```
Comment (jelletreep) [inline bioacoustics/feature_extraction/README.md:38]:
```suggestion
To extract audio features for CNN classifier, .wav files are converted to Log-mel Spectrograms using [librosa](https://zenodo.org/badge/latestdoi/6309729). 
```

File: bioacoustics/feature_extraction/README.md [hunk 2]
```diff
@@ -29,8 +29,35 @@ Use shell script `run.sh` to start `main.py` from the command line. The followin
...
 
-## sndfile library
+### sndfile library
 If you get an error saying something about a 'snd_file' dependency on an ubuntu machine, this can be fixed by installing the following C library:
 ```
 sudo apt-get install libsndfile-dev
 ```
+## Feature extraction for Convolutional Neural Network (CNN)
+To extract audio features for CNN classifier, .wav files are converted to Log-mel Spectrograms using [librosa](https://zenodo.org/badge/latestdoi/6309729) library. 
+Log- Melspectrograms had the best results in [[1]](#ref). As a future work we can try others such as Log-Spectrograms, and Gammatone-Spectrogram.
```
Comment (jelletreep) [inline bioacoustics/feature_extraction/README.md:39]:
```suggestion
Log-Melspectrograms had the best results in [[1]](#ref). As a future work we can try others such as Log-Spectrograms, and Gammatone-Spectrograms.
```

File: bioacoustics/feature_extraction/README.md [hunk 3]
```diff
@@ -29,8 +29,35 @@ Use shell script `run.sh` to start `main.py` from the command line. The followin
...
+
+In this process, first we apply butter_bandpass filter to select frequencies among 100, 2000 hz. Then the short time Fourier transform (STFT) is applied on time-domain waveforms to calculate spectrograms.
```
Comment (jelletreep) [inline bioacoustics/feature_extraction/README.md:41]:
```suggestion
In this study, first we apply a [Butterworth (bandpass) filter](https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.butter.html) to filter frequencies between 100 and 2000 hz for further processing. Then the short time Fourier transform (STFT) is applied to create spectrograms. 
```

File: bioacoustics/feature_extraction/README.md [hunk 4]
```diff
@@ -29,8 +29,35 @@ Use shell script `run.sh` to start `main.py` from the command line. The followin
...
 If you get an error saying something about a 'snd_file' dependency on an ubuntu machine, this can be fixed by installing the following C library:
 ```
 sudo apt-get install libsndfile-dev
 ```
+## Feature extraction for Convolutional Neural Network (CNN)
+To extract audio features for CNN classifier, .wav files are converted to Log-mel Spectrograms using [librosa](https://zenodo.org/badge/latestdoi/6309729) library. 
+Log- Melspectrograms had the best results in [[1]](#ref). As a future work we can try others such as Log-Spectrograms, and Gammatone-Spectrogram.
+
+In this process, first we apply butter_bandpass filter to select frequencies among 100, 2000 hz. Then the short time Fourier transform (STFT) is applied on time-domain waveforms to calculate spectrograms. 
+Then mel filter banks are applied on the spectrograms followed by a logarithmic operation to extract log mel spectrograms.
```
Comment (jelletreep) [inline bioacoustics/feature_extraction/README.md:42]:
```suggestion
Then we convert the spectrograms to MFCC (Mel-Frequency Cepstral coefficient) representation, which is often done for speech processing (Find more info [here](https://speechprocessingbook.aalto.fi/Representations/Melcepstrum.html)).
```

**Review by jelletreep — CHANGES_REQUESTED** [review]
Hi Parisa,
Looks good! I added some suggestions for the README, if you agree you can click commit suggestion

**Comment by parisa-zahedi** [comment]
thanks Jelle, for the comments! I committed all!

**Review by jelletreep — APPROVED** [review]

**Pull request merged**